client/blocks/login/one-tap-auth-loader.tsx

```diff
@@ -0,0 +1,13 @@
+import { Step } from '@automattic/onboarding';
+import { ProgressBar } from '@wordpress/components';
+
+import './one-tap-auth-loader.scss';
+
+export default function OneTapAuthLoader() {
+	return (
+		<div className="one-tap-auth-loader">
+			<Step.TopBar className="one-tap-auth-loader__top-bar" compactLogo />
```

Comment:
Is this valid? I don't see the class name prop (and neither want to 😂 ) in the TopBar component.